web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -113,15 +125,18 @@ const TeamReviews = ({ onBack, periodId }) => {
   const history = useHistory();
   const location = useLocation();
 
+  const [approvalMode, setApprovalMode] = useState(false);
```

Comment:
This is true when the user is a supervisor AND the status of the review period is AWAITING_APPROVAL.